src/include/stir/recon_buildblock/BackProjectorByBin.h

```diff
@@ -111,14 +112,42 @@ class BackProjectorByBin :
 		   const RelatedViewgrams<float>&,
 		   const int min_axial_pos_num, const int max_axial_pos_num,
 		   const int min_tangential_pos_num, const int max_tangential_pos_num);
+#endif
+ //! project whole proj_data into the volume
+ /*! it overwrites the data already present in the volume */
+ virtual void back_project(const ProjData&, int subset_num = 0, int num_subsets = 1);
+
+ /*! \brief projects the viewgrams into the volume
+  it adds to the data already present in the volume.*/
+void back_project(const RelatedViewgrams<float>&);
+
+ /*! \brief projects the specified range of the viewgrams into the volume
+  it adds to the data already present in the volume.*/
+void back_project(const RelatedViewgrams<float>&,
+          const int min_axial_pos_num, const int max_axial_pos_num);
+
+ /*! \brief projects the specified range of the viewgrams into the volume
+   it adds to the data already present in the volume.*/
+void back_project(const RelatedViewgrams<float>&,
+          const int min_axial_pos_num, const int max_axial_pos_num,
+          const int min_tangential_pos_num, const int max_tangential_pos_num);
 
+ /*! \brief tell the back projector to start accumulating into a new image*/
```

Comment:
```suggestion
 /*! \brief tell the back projector to start accumulating into a new target
 
 This function has to be called before any back-projection is initiated. 
 */
```
however, this isn't accurate, as it doesn't apply for the old versions (and in particular for the one old function that we kept).